### PR TITLE
#775 added the labels field in the add/edit viewcontroller

### PR DESCRIPTION
**Pull request by aleene**
## PR Description
Added a labels field.

Type of Changes 

- [ ] Fixes Issue #775

### CODE REVIEW

**Comment by aleene** [comment]
I increased the schemaVersion to 34, as a new field is introduced

**Comment by aleene** [comment]
A remaining issue that that the edited field is not uploaded.

**Comment by teolemon** [comment]
Noted that it's still a draft.
I've added @philippeauriach who's familiar with the editing part.

File: Sources/ViewControllers/Products/Detail/ProductDetailViewController.swift [hunk 1]
```diff
@@ -61,7 +61,7 @@ class ProductDetailViewController: ButtonBarPagerTabStripViewController, DataMan
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        //TODO: Answers.logContentView(withName: "Product's detail", contentType: "product_detail", contentId: product.barcode, customAttributes: ["product_name": product.name ?? ""])
+        // Answers.logContentView(withName: "Product's detail", contentType: "product_detail", contentId: product.barcode, customAttributes: ["product_name": product.name ?? ""])
```
Comment (philippeauriach) [inline Sources/ViewControllers/Products/Detail/ProductDetailViewController.swift]: This is not part of this current PR, but as it has been touched: Answers is part of Crashlytics, which is not used in the app anymore. I think we added the same behaviour using sentry breadcrumbs, we should check if we did and then remove this comment

Comment (aleene) [reply Sources/ViewControllers/Products/Detail/ProductDetailViewController.swift]: I always try to remove some of the swiftlint(?) warnings.

Comment (philippeauriach) [reply Sources/ViewControllers/Products/Detail/ProductDetailViewController.swift]: But TODOs are probably here for a reason ?

Comment (aleene) [reply Sources/ViewControllers/Products/Detail/ProductDetailViewController.swift]: true

**Comment by teolemon** [comment]
@aleene @philippeauriach good to merge ?

**Comment by aleene** [comment]
Seems to work for me.

**Comment by aleene** [comment]
I will merge this in #798

**Pull request closed without merging**